src/BlockchainReader/Bitcoin.test.ts

```diff
@@ -25,7 +25,7 @@ describe('Bitcoin.blockToPoetAnchors', async assert => {
     expected: true,
   })
 
-  const ipfsDirectoryHashes = ['QmaKtaPqfWvss2ndhGdcLRc9uGcLvUUyuZeGcrDZWN58Zi']
+  const ipfsDirectoryHashes = ['9tmyge2RGfyymosruzDgPjdHNXpFmp5BqY5fXNqskw4PitFPuZaKQVcdcodDqPE']
```

Comment:
@lautarodragan I'm trying to figure out why I have to change the value here.

Reply:
I think it's because this is a breaking change and the tests are grabbing test data that is no longer valid.

```ts
import * as TestBlock from './TestData/block-0000000070746b06bbec07a7cd35e0c6d47bfa4e2544a6a1d2aa6efc49d47679.json'

const poetAnchors = blockToPoetAnchors(TestBlock as any) // as any: see footer note
const anchorsIpfsHashes = poetAnchors.map(poetAnchor => poetAnchor.ipfsDirectoryHash)
const ipfsDirectoryHashes = ['9tmyge2RGfyymosruzDgPjdHNXpFmp5BqY5fXNqskw4PitFPuZaKQVcdcodDqPE']
const anchorHasUnexpectedIpfsHash = (poetAnchor: PoetBlockAnchor) =>
  !ipfsDirectoryHashes.includes(poetAnchor.ipfsDirectoryHash)
```

Then these two tests compare the new code agains the old test data, which fails:

```ts
    assert({
      given,
      should: 'all returned elements should have an expected IPFS Directory Hash',
      actual: poetAnchors.find(anchorHasUnexpectedIpfsHash),
      expected: undefined,
    })
// ...
    assert({
      given,
      should: 'all expected IPFS Directory hashes should be in the returned anchors',
      actual: equals(anchorsIpfsHashes.sort(localeCompare), ipfsDirectoryHashes.sort(localeCompare)),
      expected: true,
    })
```

So, TDD-wise, you shouldn't be changing this line of code and the tests should fail, initially.

Then, to fix the failing tests, you'll need new test data.

I grabbed this test data originally by running new code on testnet and downloading the block in which my test transactions got confirmed. 

The problem with this approach is that we could be introducing a bug and kind of "forcing" the tests to pass, so it requires paying close attention to the changes (to both the PR creator and the reviewers). I'm open to better alternatives, but this change is pretty small so downloading a new testnet block can be enough.

Reply:
yes, this is my concern. I'm trying to understand all around this test